install/gcp/up.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -u
+set -x
```

Comment:
Best to add `set -e` to exit on error?

Reply:
I removed it on purpose so I don't have to add a ton of if checks for gke cluster and nodepool creation. I can add it back and add if statements.

Reply:
I've added it back